src/hadoop-resource-manager/deploy/hadoop-resource-manager-configuration/yarn-site.xml.template

```diff
@@ -313,35 +313,40 @@
 
   <property>
     <description>
-    The duration (in ms) the YARN client waits for an expected state change
-    to occur.  -1 means unlimited wait time.
+      The duration (in ms) the YARN client waits for an expected state change
+      to occur.  -1 means unlimited wait time.
     </description>
     <name>yarn.client.application-client-protocol.poll-timeout-ms</name>
     <value>900000</value>
   </property>
 
   <property>
-    <description>The minimum allocation for every container request at the RM,
+    <description>
+      The minimum allocation for every container request at the RM,
       in terms of GPUs. Requests lower than this will throw a
-      InvalidResourceRequestException.</description>
+      InvalidResourceRequestException.
+    </description>
     <name>yarn.scheduler.minimum-allocation-gpus</name>
     <value>0</value>
   </property>
 
   <property>
-    <description>The maximum allocation for every container request at the RM,
+    <description>
+      The maximum allocation for every container request at the RM,
       in terms of GPUs. Requests higher than this will throw a
-      InvalidResourceRequestException.</description>
+      InvalidResourceRequestException.
+    </description>
     <name>yarn.scheduler.maximum-allocation-gpus</name>
-    <value>8</value>
+    <value>16</value>
```

Comment:
A larger value such as 64? which is the maximum supported value. The same suggestion on cpu & memory.

Reply:
We can treat DGX-2 as the upper bond of resources.
You won't find any machines have more than 16 GPU cards except for customized mother boards.

Reply:
OK, keep it before a configurition